src/MessagePack.UnityClient/Assets/Scripts/MessagePack/IFormatterResolver.cs

```diff
@@ -76,9 +81,17 @@ public static object GetFormatterDynamic(this IFormatterResolver resolver, Type
                 throw new ArgumentNullException(nameof(type));
             }
 
-            MethodInfo methodInfo = typeof(IFormatterResolver).GetRuntimeMethod(nameof(IFormatterResolver.GetFormatter), Type.EmptyTypes);
+            var formatterGetter = FormatterGetters.GetOrAdd(type, t =>
+            {
+                MethodInfo methodInfo = typeof(IFormatterResolver).GetRuntimeMethod(nameof(IFormatterResolver.GetFormatter), Type.EmptyTypes);
```

Comment:
This could be done in a static field once and reused.

Reply:
agree